source/appModules/eclipse.py

```diff
@@ -52,7 +55,8 @@ def script_closeAutocompleter(self, gesture):
 	@script(
 		# Translators: Input help mode message for the 'read documentation script
 		description = _("Tries to read documentation for the selected autocompletion item."),

```

Comment:
Linting: remove spaces around = when used to define the keyword parameter of a function.

Also modified the constant for the category name.

```suggestion
		description=_("Tries to read documentation for the selected autocompletion item."),
```